archive-news.php

```diff
@@ -0,0 +1,49 @@
+<?php // archive-news.php ?>
+
+<?php get_header(); ?>
+
+	<?php
+
+		// recent news
+		$latest_news = new query_loop([
+			'post_type' => 'news',
+			'posts_per_page' => -1
+		]);
+
+	?>
+
+	<div class="wrapper / news__container">
+
+		<div class="band band--thin">
+
+			<a href="<?php echo get_permalink(get_option('page_for_posts')); ?>" class="text-button button--icon">
+				<svg><use xlink:href="#icon-arrow-left"></svg><?php _e('Back to latest', 'ocp'); ?>
```

Comment:
I'd say this probably isn't worth it. In HTML5 the trailing slash on a "self-closing element" is ignored anyway. I guess the real question here is consistency, and we should probably start to move towards no slashes throughout.